COVID.Label.Exposure.Child.R

```diff
@@ -81,7 +81,7 @@ label(CHILD$covid_background_health___12)="Has a health professional ever told y
 label(CHILD$covid_background_health___13)="Has a health professional ever told you that you had any of the following health conditions? (choice=Epilepsy or seizures)"
 label(CHILD$covid_background_health___14)="Has a health professional ever told you that you had any of the following health conditions? (choice=Serious stomach or bowel problems)"
 label(CHILD$covid_background_health___15)="Has a health professional ever told you that you had any of the following health conditions? (choice=Serious acne or skin problems)"
-label(CHILD$covid_background_health___16)="Has a health professional ever told you that you had any of the following health conditions? (choice=Emotion or mental health problems (ex. Depression or Anxiety))"
+label(CHILD$covid_background_health___15)="Has a health professional ever told you that you had any of the following health conditions? (choice=Emotion or mental health problems (ex. Depression or Anxiety))"
```

Comment:
There is now 2 variables called covid_background_health___15. What was fixed here?